pandas/tests/plotting/test_xlim_ylim_dtype.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+import pandas as pd
+from datetime import datetime as dt
+import numpy as np
```

Comment:
could you please use `isort` to resort the imports, otherwise, I think black check will fail.

Also please don't use `import pandas as pd`.

Reply:
I have formatted the import list with `isort`

Reply:
you probably need to import matplotlib in order to fix the failed tests

Reply:
I have added an import statement

Reply:
@regmibijay I see that matplotlib import is still missing.
Can you double check that?

Reply:
I am having trouble understanding
` ModuleNotFoundError: No module named 'matplotlib'`  in the CI tests, I `import matplotlib ` normally. Is there something very obvious I am missing?

Reply:
@regmibijay 
From the current test module, you actually do not need matplotlib to be imported.

The reason the CI fails is that because matplotlib is simply not installed for all pipelines.
See ``pandas/ci/deps`` for dependencies.

In order to ensure that your test module is skipped in the pipelines, which do not have matplotlib installed,
then you need to add ``pytest.imporotorskip`` to the top and probably mark the tests as ``slow``.

```
pytest.importorskip("matplotlib")
pytestmark = pytest.mark.slow
```

See ``pandas/tests/plotting/test_style.py``, for instance.

Reply:
I just pushed new commit with changes suggested by you.